css/style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  background-color: #EEE;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+h1 {
+  color: red;
+  font-family: cursive, sans-serif;
+  text-align: center;
+  font-size: 32px;
+}
+
+h2 {
+  color: maroon;
+  font-family: cursive, sans-serif;
+  text-align: center;
+}
+
+p {
+  color: black;
+  font-family: fantasy, sans-serif;
+  text-align: left;
+}
+
+img {
+  border: 10px solid;
+  padding: 5px;
+  margin: 0px 60px 10px 60px;
+  height: 400px;
+  float: right;
+}
+
+.portfolio {
+  border: 0px;
+  margin: 0px 40px 10px 40px;
+  padding: 0px;
+  height: 500px;
+  float: none;
+}
+
+img.portrait {
```

Comment:
Combining tags and classes is not necessary and drives up the specificity without need.  Please read this article on specificity (https://css-tricks.com/specifics-on-css-specificity/) and BEM (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/).  BEM is the most popular of the modular strategies to handle styling with classes (which is the industry best practice).  It is used by Google, AirBnB, etc, and is my go to way of organizing CSS.